web/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="dib pa3 ma0 lh-tight">
     <section class="bg-near-white">
         <form id="queryForm" class="mw8 center lh-copy dark-gray br2 pv4 ph2 ph4-ns">
             <label class="db mt3 f6 fw6" for="multiaddr">Multiaddr</label>
-            <input class="db w-100 pa2" type="text" id="multiaddr" name="multiaddr" placeholder="/ip4/1.2.3.4/tcp/1234/p2p/QmFoo" required />
+            <input class="db w-100 pa2" type="text" id="multiaddr" name="multiaddr" placeholder="/p2p/QmFoo" required />

```

Comment:
I figured searching for PeerIDs would be more common here, but leaving in a more complete multiaddr seems like fair game as well if it's more likely to encourage people to just think of this as a multiaddr field

Reply:
I think the shortened version is nicer as a newer user might have a greater chance of figuring out how to use it, where a more experienced user should be able to infer that full multiaddrs are supported if they either just try it out, or read the lower text referring to the advanced menu.